### ISSUE TITLE
Headings are not structured - SiteImprove - Grad SROP

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9100]
# Problem

The project title (https://grad.uiowa.edu/node/add/scholar) for the SROP scholar content type is set to an H3 and the name of the scholar is an H1 which is throwing an accessibility heading order error. 
 
[https://my2.siteimprove.com/Inspector/998426/A11Y/Page?pageId=78156286932&impmd=5474C4B537EC8F6FA1344F9DB1AEDA5C&issue=cantTell+failed&wcag=twopointtwo#/sia-r53/failed/eVAwUEF2d2E=;ZmQzMjg5NzY=](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmy2.siteimprove.com%2FInspector%2F998426%2FA11Y%2FPage%3FpageId%3D78156286932%26impmd%3D5474C4B537EC8F6FA1344F9DB1AEDA5C%26issue%3DcantTell%2Bfailed%26wcag%3Dtwopointtwo%23%2Fsia-r53%2Ffailed%2FeVAwUEF2d2E%3D%3BZmQzMjg5NzY%3D&data=05%7C02%7Cits-web%40uiowa.edu%7Caa1d612f2bc7406a130e08dddb662457%7C1bc445959aba4fc3b8ec7b94a5586fdc%7C1%7C0%7C638907953941287097%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=ehp3kJyo1sJHxeAJl9x2ZFYIgsztHKUAS9gx%2FAFz124%3D&reserved=0)
